### PR TITLE
crimson/os/seastore/seastore_types: cleanups

### PR DESCRIPTION
**Pull request by cyx1231st**
## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by cyx1231st** [comment]
enkins test docs

**Comment by cyx1231st** [comment]
jenkins test docs

File: src/crimson/os/seastore/seastore_types.cc [hunk 1]
```diff
@@ -176,16 +201,16 @@ std::ostream &operator<<(std::ostream &out, const paddr_list_t &rhs)
   return out << ']';
 }
 
-std::ostream &operator<<(std::ostream &lhs, const delta_info_t &rhs)
+std::ostream &operator<<(std::ostream& out, const delta_info_t& delta)
```
Comment (athanatos) [inline src/crimson/os/seastore/seastore_types.cc:204]: std::ostream &out, const delta_info_t &delta

File: src/crimson/os/seastore/seastore_types.h [hunk 2]
```diff
@@ -914,11 +926,9 @@ struct delta_info_t {
       bl == rhs.bl
     );
   }
-
-  friend std::ostream &operator<<(std::ostream &lhs, const delta_info_t &rhs);
 };
 
-std::ostream &operator<<(std::ostream &lhs, const delta_info_t &rhs);
+std::ostream& operator<<(std::ostream& out, const delta_info_t& delta);
```
Comment (athanatos) [inline src/crimson/os/seastore/seastore_types.h:931]: See above

**Review by athanatos — CHANGES_REQUESTED** [review]
LGTM other than the & placement

File: src/crimson/os/seastore/seastore_types.cc (hunk 1 above)
Comment (cyx1231st) [reply src/crimson/os/seastore/seastore_types.cc]: done, and fixed the others

**Comment by cyx1231st** [comment]
271 - run-tox-alerts-lint (Failed)

**Comment by cyx1231st** [comment]
jenkins test make check

File: src/crimson/os/seastore/seastore_types.cc (hunk 1 above)
Comment (cyx1231st) [reply src/crimson/os/seastore/seastore_types.cc]: I've got opposite placement suggestions previously.

Do we have a strong preference about & placement in Ceph? Or it is OK as long as the style is consistent in the same file?

**Comment by cyx1231st** [comment]
jenkins test api

File: src/crimson/os/seastore/seastore_types.cc (hunk 1 above)
Comment (athanatos) [reply src/crimson/os/seastore/seastore_types.cc]: Most code has been written with the &/* on the variable name, so let's just stick with that.

**Comment by cyx1231st** [comment]
@athanatos do you have any further concerns about the changes?

**Review by athanatos — APPROVED** [review]

**Comment by athanatos** [comment]
Sorry, forgot to take another look after you updated it.

**Pull request merged**